ml_pipeline_engine/types.py

```diff
@@ -63,93 +57,22 @@ def next_iteration(self, data: t.Optional[AdditionalDataT]) -> Recurrent:
         """
 
 
-class NodeProtocol(t.Protocol):
+class NodeBase(RetryProtocol, TagProtocol, t.Protocol[NodeResultT]):
     """
-    Узел графа модели
+    Basic node interface
     """
-
-    RUN_METHOD_ALIASES = (
-        'process',
-        'extract',
-        'collect',
-        'vectorize',
-        'predict',
-    )
+    RUN_METHOD_ALIAS = 'process'
```

Comment:
I still have a question for this method. If we remove the ability to use different names in subclasses, I guess that we actually don't need to remain logic with a method's name as a string. We can just leave the abstract method "process" and use it. It will help understand codebase much faster and less painfully because the method is more explicit than the string name of the method

Reply:
ok, I've removed the constant, but I decided not to touch any other method dependencies; I suggest we take a look and decide what to eliminate from annotations in the next iteration

diff: edd533b8c0fdfbfb6a85a7946d2a2f3c0f96a96d